image_tools_py/src/burger_py.py

```diff
@@ -0,0 +1,107 @@
+
+
+class Burger(object):
+    def __init__(self):
+        burger_png = base64.b64decode(BURGER_BASE64)
+        np_img = numpy.fromstring(burger_png, dtype=numpy.uint8)
+        self.burger_template = cv2.imdecode(np_img, cv2.IMREAD_COLOR)
+
+        # We flood fill the burger template with "1,1,1" (BGR) so that we can
+        # remove the borders during rendering.
+        h, w = self.burger_template.shape[:2]
+        mask = numpy.zeros((h + 2, w + 2), numpy.uint8)
+        cv2.floodFill(self.burger_template, mask, (1, 1), (1, 1, 1))
+
+        random.seed()
+
+        self.width = 0
+        self.height = 0
+        self.num_burgers = 0
+        self.burger_list = []
+
+    def render_burger(self, width, height):
```

Comment:
Agreed.  However, it is called "render_burger" in the C++ example.  I don't really care either way.